specification/appcomplianceautomation/AppComplianceAutomation.Management/resources/EvidenceResource.tsp

```diff
@@ -0,0 +1,122 @@
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@azure-tools/typespec-client-generator-core";
+import "@typespec/openapi";
+import "@typespec/rest";
+import "../models/models.tsp";
+import "./ReportResource.tsp";
+
+using TypeSpec.Rest;
+using Azure.ResourceManager;
+using TypeSpec.Http;
+using OpenAPI;
+using Azure.ClientGenerator.Core;
+
+namespace Microsoft.AppComplianceAutomation;
+/**
+ * A class represent an AppComplianceAutomation evidence resource.
+ */
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance" "For backward compatibility"
+#suppress "@azure-tools/typespec-azure-resource-manager/arm-resource-invalid-envelope-property" "For backward compatibility"
+@parentResource(ReportResource)
+@includeInapplicableMetadataInPayload(false)
+@Azure.ResourceManager.Private.armResourceInternal(EvidenceProperties)
+model EvidenceResource extends Azure.ResourceManager.Foundations.ProxyResource {
```

Comment:
since this is the first ga, i suggest to use standard `ProxyResource` instead of keep full backward compatibility.